woo-razorpay.php

```diff
@@ -3509,7 +3510,6 @@ function disable_coupon_field_on_cart($enabled)
 if(is1ccEnabled())
 {
     add_filter('woocommerce_coupons_enabled', 'disable_coupon_field_on_cart');
-    add_action('woocommerce_cart_updated', 'enqueueScriptsFor1cc', 10);
```

Comment:
@abdulwahidsharief As we removed this hook, can you please verify the 1cc buttons are properly rendering on cart update as well?